CHANGELOG.md

```diff
@@ -4,11 +4,35 @@
 
 ### Fixes
 
+## 3.11.0 (Feature release)
+
+### New features
+
+#### Allow users to accept or reject cookies with a cookie banner
+
+You can now [use cookie banners](https://design-system.service.gov.uk/components/cookie-banner/) to allow users to accept or reject cookies which are not essential for making your service work.
+This was added in pull request #?: Add cookie banner.
+
+#### Group buttons and links
+
+You can now use a button group when you place two or more buttons together.
+Any links you include within a button group now line up visually with the buttons.
+Read more about [button groups on the Design System’s button page](https://design-system.service.gov.uk/components/button/#grouping-buttons).
```

Comment:
Do we need to check that the link works?

Reply:
Looks like it'll be fine, linked to in https://deploy-preview-1472--govuk-design-system-preview.netlify.app/components/button/#grouping-buttons